#### Common parameters
float64 leaf_size
float64 normal_radius_ratio_leaf
float64 feature_radius_ratio_leaf
uint32 	correspondence_randomness
float64 similarity_threshold
uint32 	max_iterations
float64 max_correspondence_distance_ratio_leaf
float64 inlier_fraction

## ICP
uint32 	icp_max_iterations
float64 icp_max_correspondence_distance_ratio_leaf

# Alignment Prerejective
float64 min_score
float64 min_score_threshold




